### DISCUSSION TITLE
is there a z offset feature?

### DISCUSSION THREAD

**Discussion by CloudViaLattea** [jneilliii/OctoPrint-BedLevelVisualizer#527] [Q&A]
Hello, 
Reading though all the information I can but not seeing anywhere that there is a setting to set a Z offset from the probes height and the nozzles height difference is there a setting that does this or if not this is something I would like to suggest to make it a more all inclusive package. 

any info regarding this is much appreciated as I know there are other addons that do this as well just wonder if im missing something with this one.

Thanks 

EDIT: have been using a workaround in adding a line of gcode to end of settings for setting the offset just was unsure if the was an alternate way

**Comment by jneilliii** [comment]
The relative z offset option will move the lowest point in the returned mesh to be "0".
